packages/eslint-config-airbnb/rules/es6.js

```diff
@@ -83,6 +86,22 @@ module.exports = {
     'template-curly-spacing': 2,
     // enforce spacing around the * in yield* expressions
     // http://eslint.org/docs/rules/yield-star-spacing
-    'yield-star-spacing': [2, 'after']
+    'yield-star-spacing': [2, 'after'],
+    // disallow invalid exports, e.g. multiple defaults
+    // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/export.md
+    'import/export': 2,
+    // prevents `import * from 'foo';`
```

Comment:
can we add the rule link to https://github.com/airbnb/javascript#modules--no-wildcard ?